source/user-manual/reference/tools/keystore-tool.rst

```diff
@@ -0,0 +1,33 @@
+.. Copyright (C) 2015, Wazuh, Inc.
+
+.. meta::
+  :description: Tool where sensitive configuration data can be securely stored, including any information that the Wazuh server daemons/tools need for their work. 
+
+.. _keystore-tool:
+
+keystore-tool
+=============
+
+The keystore-tool increases the security of sensitive information, storing in it any information that the Wazuh server requires for its correct operation.
```

Comment:
Wazuh server -> Wazuh manager

Reply:
Done in [Wazuh components are named appropriately](https://github.com/wazuh/wazuh-documentation/pull/7000/commits/ef178ca855e5c05eca6c5c8705a7e54e2199f3bb) commit.